src/main/java/hudson/plugins/git/extensions/impl/PreBuildMerge.java

```diff
@@ -55,6 +55,15 @@ public UserMergeOptions getOptions() {
 
     @Override
     public Revision decorateRevisionToBuild(GitSCM scm, Run<?, ?> build, GitClient git, TaskListener listener, Revision marked, Revision rev) throws GitException, IOException, InterruptedException {
+
+        if ( build == null){
```

Comment:
Consistent formatting helps me as a reviewer.  Optional change, but preferred
```suggestion
        if (build == null) {
```